src/OpenSSL/SSL.py

```diff
@@ -161,6 +164,47 @@ class SysCallError(Error):
     pass
 
 
+def _make_buffer_checker(cffi_version_string):
+    """
+    Returns a callable that converts a sendable buffer object into something
+    that can be sent by OpenSSL using CFFI. Has different supported types
+    depending on the CFFI version.
+
+    :param cffi_version_string: The version of CFFI in use, from
+        ffi.__version__.
+    """
+    cffi_version = parse(cffi_version_string)
```

Comment:
A quick check shows that cffi has had a pre-parsed version tuple at `cffi.__version_info__` for some time (I checked 1.0.0 and it had it). That might be nicer than pulling in a dependency on `packaging`?